extensions/amp-story/amp-story.md

```diff
@@ -362,6 +374,48 @@ Usage: `<amp-story ... supports-landscape>...</amp-story>`
   </amp-anim>
 </figure>
 
+### Live story
+
+Using the `live-story` attribute, you can append new pages to a story for users to see in real-time. If the user is on the last page it will show a notification saying there are new pages and the progress bar will be updated.
+
+The core use cases for this are: coverage for breaking news or live events where the user can stay on to see new pages as they come in. Common examples are award shows, sporting events, and elections.
```

Comment:
Update to:

Core use cases for `live-story` include coverage for breaking news or live events, enabling real-time updates to users without exiting the story. Award shows, sporting events and elections are some examples.